vlc-android/res/values/strings.xml

```diff
@@ -154,6 +154,9 @@
     <string name="navmenu">navigation menu</string>
     <string name="advanced">advanced options</string>
     <string name="audio_boost_warning">Slide up again to go beyond 100%</string>
+    <string name="locked_in_portrait_mode">Locked in portrait mode</string>
```

Comment:
Maybe we should add "(long click to reset to previous mode)"

Because feature would be hidden otherwise, but it makes the text a bit too long.

Reply:
Maybe we should add sizes for void showInfo(String text, int duration) 
Like, small, medium, large (or pass directly the float size like the duration parameter)
and use small for longer messages.